abi/codecForAddress.go

```diff
@@ -5,12 +5,13 @@ import (
 	"io"
 )
 
-type codecForAddress struct {
-	pubKeyLength int
+// AddressValue is a wrapper for an address
```

Comment:
File will be renamed in a separate PR (to avoid a bigger delta).

Reply:
The same for other files. E.g. `codec for struct`, `codec for enum` will become `structValue.go`, `enumValue.go`.